docs/example-apps/rails.md

```diff
@@ -71,8 +56,42 @@ And you can deploy your application to production
 convox deploy
 ```
 
+You need to define the secret key for your production application
+
+```
+convox env set SECRET_KEY_BASE="$(rails secret)"
+```
+
 Finally you can retrieve the URL from your production application with
 
 ```bash
 convox services
 ```
+
```

Comment:
Hmm, if this is a step-by-step guide, I'm not convinced that leaving the explanations to the end helps the end-user understand what's happening 🤔

Reply:
👍🏻